connectors/src/connectors/microsoft/temporal/activities.ts

```diff
@@ -402,11 +448,13 @@ export async function syncFiles({
 
 export async function syncDeltaForRootNode({
   connectorId,
-  rootNodeId,
+  driveId,
+  parentIds,
```

Comment:
nit: would stick with `rootNodeIds` as name since that's what they are (parents could be non-root nodes, and somewhat suggests we're gonna only act on children, while here we update the root nodes too, as well as the children)